### PR TITLE
Side bar changes for better clarity and presentation

### PR DESCRIPTION
**Pull request by gordon-fish**
Display "Development release" to side bar (under "Module version")
with matching text color to the release text in bread brumbs area
for dev ("maturity": "developer") releases. This is to make it
more clear that a release (especially when the version only has
an underscore, and no "TRIAL" or other such clues) is in fact a
development release.

When there is only one version for a distribution, display a
disabled drop down with "No other versions" text in place of,
and matching the style of, the "Jump to version" drop down, in
the side bar.

Hide "Latest version" link in side bar when there are no stable
("maturity": "released") releases, which would result in a 404
if followed.

### CODE REVIEW

File: root/inc/release-tools.html [hunk 1]
```diff
@@ -64,13 +70,20 @@
...
 </ul>
-<%- END %>
+<%- END;
+stable = [];
+FOREACH version IN versions;
+    IF version.maturity == 'released';
+        stable.push(version);
+        NEXT;
+    END;
+END %>
```
Comment (oalders) [inline root/inc/release-tools.html]: This might be better off as a check to see if there's a release with `release.status == 'latest'` and then short-circuit at that point.  If there's no release with a `latest` status, then the `Latest version` link won't work.

File: root/pod.html [hunk 2]
```diff
@@ -27,6 +27,11 @@
       Module version: <span itemprop="softwareVersion"><% documented_module.version | html %></span>
     </li>
     <% END %>
+    <% IF release.maturity == 'developer' %>
+    <li>
+      <b style="color: #ff4500">Development release</b>
```
Comment (oalders) [inline root/pod.html]: Noted that this will be addressed in a different PR.

File: root/inc/release-tools.html [hunk 3]
```diff
@@ -56,6 +56,12 @@
         <%- PROCESS version_options %>
     </select>
 </li>
+<%- ELSE -%>
+<li>
+    <select class="form-control tool-bar-form" style="font-style: italic" disabled="disabled">
+        <option>No other versions</option>
```
Comment (oalders) [inline root/inc/release-tools.html]: 👍

File: root/inc/release-tools.html (hunk 1 above)
Comment (gordon-fish) [reply root/inc/release-tools.html]: Good point. I'll make that adjustment. Thanks.

Comment (gordon-fish) [reply root/inc/release-tools.html]: I've updated the pull request with that change.

Comment (gordon-fish) [reply root/inc/release-tools.html]: https://github.com/metacpan/metacpan-web/blob/377054534562a2989d1cc48dd125b35a9273644b/root/inc/release-tools.html#L73-L80

File: root/pod.html (hunk 2 above)
Comment (gordon-fish) [reply root/pod.html:32]: Done :) #2284
